patroni/config.py

```diff
@@ -220,6 +220,15 @@ def _popenv(name):
             if value:
                 ret[param] = value
 
+        def _fix_log_env(name, oldname):
+            value = _popenv(oldname)
+            name = Config.PATRONI_ENV_PREFIX + 'LOG_' + name.upper()
+            if value and name not in os.environ:
```

Comment:
can you pls swap conditions as in 
`name not in os.environ and value :` or put explicit parentheses around `name not ..`?
imo makes it a bit easier to read (no need to remember precedence of logical operations in that case)

Reply:
The code is clean and readable here:
1. `not in` is the standard python operator, it is **not** a combination of two operators
2. My internal branch predictor tells me, that most likely the value is `None` here, and therefore it should be the first in the if statement.